<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entertaining Mines</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/entertaining-mines.svg</normaloff>:/icons/entertaining-mines.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="tStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="MainScreen" name="mainScreen"/>
      <widget class="GameScreen" name="gameScreen"/>
      <widget class="SettingsScreen" name="settingsScreen"/>
      <widget class="MainOnlineScreen" name="onlineScreen"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainScreen</class>
   <extends>QWidget</extends>
   <header>screens/mainscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameScreen</class>
   <extends>QWidget</extends>
   <header>screens/gamescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsScreen</class>
   <extends>QWidget</extends>
   <header>screens/settingsscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainOnlineScreen</class>
   <extends>QWidget</extends>
   <header>online/mainonlinescreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
